app/templates/main/download-confirmation.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{%- from "govuk_frontend_jinja/components/accordion/macro.html" import govukAccordion -%}
+{%- from 'govuk_frontend_jinja/components/checkboxes/macro.html' import govukCheckboxes -%}
+{%- from 'govuk_frontend_jinja/components/select/macro.html' import govukSelect -%}
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+{%- from 'govuk_frontend_jinja/components/label/macro.html' import govukLabel -%}
+{%- from "outcomeCheckboxes.html" import outcomeCheckboxItems -%}
+{%- from "checkboxes.html" import checkboxItems -%}
+{%- from "select.html" import selectItems -%}
+
+{% block beforeContent %}
+{{ super() }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1 class="govuk-panel__title">
+                Request received
+            </h1>
+        </div>
+        <h2 class="govuk-heading-m govuk-!-margin-top-7">
+            What happens next
+        </h2>
+        <p class="govuk-body">
+            We will email a link to
+            <span class="govuk-!-font-weight-bold">
+            {{user_email}}
+            </span>.
+        </p>
+        <p class="govuk-body">
+            This may take anything up to 5 minutes to come through to your inbox.
+        </p>
+        <h2 class="govuk-heading-m">
+            If you’ve not recieved the email
+        </h2>
+        <p class="govuk-body">
+            If you cannot see the email in your inbox, check your junk or spam folder.
+            If you’re still having problems,
+            <a href="/download">
+                request a new link to download your data
+            </a>
+            or email us at
+            <a href="mailto:fsd.support@levellingup.gov.uk">
+                fsd.support@levellingup.gov.uk
+            </a>.
```

Comment:
Because this on a new line, we end up with a space between the email address and the full stop. Which looks a bit awkward. The nittest of picks, but if we can online this it'll look better.

Can have this as a single line:
`or email us at <a class="govuk-link govuk-link--no-visited-state" href="mailto:fsd.support@levellingup.gov.uk">fsd.support@levellingup.gov.uk</a>.`

Reply:
Done!